src/main/webapp/game.js

```diff
@@ -60,15 +64,47 @@ channel.bind(PUSHER_CHAT_CHANNEL_NAME, function(data) {
 });
 
 function embedPlaylist() {
-  fetch('/game').then(response => response.json()).then((videoID) => {
-    document.getElementById("player").src = "https://www.youtube.com/embed/" + videoID;
+  fetch('/game').then(response => response.json()).then((videoIdResponse) => {
+    videoId = videoIdResponse;
+
+    var tag = document.createElement('script');
+    var firstScript = document.getElementsByTagName('script')[0];
+
+    tag.src = 'https://www.youtube.com/iframe_api';
+    firstScript.parentNode.insertBefore(tag, firstScript);
```

Comment:
What are these lines doing?

Reply:
They're retrieving the script for the youtube iframe api @pruess-google

Reply:
Got it. I'm not too familiar with JS, but could you instead include the youtube iframe API with a script in the head of game.html, like is done in line 6 of game.html for the pusher API:
https://github.com/googleinterns/step105-2020/blob/2d8488c5fba930411e7c4a95c95720c49e6b9376/src/main/webapp/game.html#L6

If you can do this, I think it would be simpler than doing it programmatically in the JS code. If not, no worries.